tests/test_difficulty.py

```
@@ -0,0 +1,414 @@
+    
+    def test_block_too_slow_increases_difficulty(self):
+        """Test that slow blocks increase difficulty."""
+        current_difficulty = 1000
+        actual_block_time = 7.0  # 2 seconds slower than target (5s)
+        
+        new_difficulty = self.adjuster.adjust(
+            current_difficulty=current_difficulty,
+            actual_block_time=actual_block_time
+        )
+        
+        self.assertGreater(new_difficulty, current_difficulty)
```

Ruff (0.15.6)

[warning] Use a regular `assert` instead of unittest-style `assertGreater`

Replace `assertGreater(...)` with `assert ...`

(PT009)